core/chains/evm/logpoller/log_poller_test.go

```diff
@@ -1525,7 +1634,15 @@ func Test_PollAndSavePersistsFinalityInBlocks(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			th := SetupTH(t, tt.useFinalityTag, tt.finalityDepth, 3, 2, 1000)
+			lpOpts := logpoller.Opts{
+				UseFinalityTag:           tt.useFinalityTag,
+				FinalityDepth:            tt.finalityDepth,
+				BackfillBatchSize:        3,
+				RpcBatchSize:             2,
+				KeepFinalizedBlocksDepth: 1000,
+				BackupPollerBlockDelay:   0,
```

Comment:
any reason for setting `BackupPollerBlockDelay` to `0` if by default it would be `0` any way? being more explicit that it's disabled?

Reply:
Good catch! Looks like I messed up a bunch of these during my last commit.  I was thinking since the default is 1000 in toml config it would just default to 1000 here, forgot it doesn't work that way ;-)